src/mumble/main.cpp

```diff
@@ -671,19 +671,29 @@ int main(int argc, char **argv) {
 }
 
 #if defined(Q_OS_WIN) && defined(QT_NO_DEBUG)
-extern void qWinMain(HINSTANCE, HINSTANCE, LPSTR, int, int &, QVector<char *> &);
-
 extern "C" __declspec(dllexport) int MumbleMain(HINSTANCE instance, HINSTANCE prevInstance, LPSTR cmdArg, int cmdShow) {
-	Q_UNUSED(cmdArg);
+	Q_UNUSED(instance)
+	Q_UNUSED(prevInstance)
+	Q_UNUSED(cmdArg)
+	Q_UNUSED(cmdShow)
+
+	int argc;
+	wchar_t **argvW = CommandLineToArgvW(GetCommandLineW(), &argc);
+	if (!argvW) {
+		return -1;
+	}
 
-	QByteArray cmdParam = QString::fromWCharArray(GetCommandLine()).toLocal8Bit();
-	int argc = 0;
+	QVector<QByteArray> argvS;
+	argvS.reserve(argc);
+
+	QVector<char *> argvV(argc, Q_NULLPTR);
+	for (int i = 0; i < argc; ++i) {
+		argvS.append(QString::fromWCharArray(argvW[i]).toLocal8Bit());
+		argvV[i] = argvS.back().data();
+	}
 
-	// qWinMain takes argv as a reference.
-	QVector<char *> argv;
-	qWinMain(instance, prevInstance, cmdParam.data(), cmdShow, argc, argv);
+	LocalFree(argvW);
 
-	int result = main(argc, argv.data());
-	return result;
+	return main(argc, argvV.data());
```

Comment:
Where is main defined?

Reply:
https://github.com/mumble-voip/mumble/blob/78a5458ccc607b0d316611c59db812966c45f5f3/src/mumble/main.cpp#L73-L671